### ISSUE TITLE
AMPATH Form Translations should take into account the form version.

### ISSUE THREAD

**Issue by ibacher** [mekomsolutions/openmrs-module-initializer#244]

**Comment by mks-d** [comment]
@ibacher is this still current our we can close this issue?

**Comment by ibacher** [comment]
This one still needs to be done. Specifically, when loading a new version of an AMPATH Form, we need to reload the translations for that form so we end up with the form resources attached to the new form.

**Comment by mks-d** [comment]
@ibacher I'm wondering why this was not needed more in our implementation(s), could you speak to it @Ruhanga?

If we could live without it until now, and that the React Form Engine supersedes the AMPATH Form Engine, then this issue may be closed no?

**Comment by ibacher** [comment]
There are two things that make this a little rare to come across:

1. Most implementations are mono-lingual and so, as long as the form works in the target language (usually English) translations aren't needed.
2. The issue only arises when you have a form available in multiple languages and release a _new_ version of the form, but with no additional translation keys needed, e.g., because you added a question that just uses the concept string.

Basically, AFAIK, the only implementation to run up against this is the ICRC, but other non-English implementations are likely to run into thi.

**Comment by mks-d** [comment]
>the only implementation to run up against this is the ICRC

In that case it would be good for @icrc-fdeniger to read through this and let us know if this issue should remain open.

**Comment by icrc-fdeniger** [comment]
I would say yes as it's not specific to ICRC but to non mono-ligual implementation.

**Comment by Ruhanga** [comment]
This issue is duplicated by [this other one](https://github.com/mekomsolutions/openmrs-module-initializer/issues/259). 

> I'm wondering why this was not needed more in our implementation(s), could you speak to it @Ruhanga?

We re-discovered it when working with the Cambodia distro.
